model/registry/nodejs.yaml

```diff
@@ -0,0 +1,20 @@
+groups:
+  - id: registry.nodejs
+    type: attribute_group
+    brief: Describes Node.js related attributes.
+    display_name: Node.js Attributes
+    prefix: nodejs
```

Comment:
we've got rid of prefix while you were working on this PR, so this should be removed and added to the attribute id

```suggestion
```